owlbot.py

```diff
@@ -42,6 +42,8 @@
 s.move(
     templated_files,
     excludes=[
+        # Need a combined LICENSE for all vendored packages.
+        "LICENSE",
```

Comment:
Just a heads up: `LICENSE` is no longer part of synthtool templates. We may need to update the post processor image if owlbot is reverting changes.

https://github.com/googleapis/synthtool/tree/master/synthtool/gcp/templates/python_library

Reply:
Got it. Yeah, next image update we can remove this line.